web/src/beta/features/Editor/StoryPanel/Block/builtin/Text/index.tsx

```diff
@@ -15,19 +17,31 @@ const TextBlock: React.FC<Props> = ({ block, isSelected, ...props }) => {
     [block?.property?.items],
   );
 
+  // Text block is very special, it will not edit values with field components
+  // from the common editor panel, but manage it by itself directly.
```

Comment:
Can you move this above the component's declaration? Seems related to the component itself, but is above the usePRopertyValueUpdate but seems unrelated

Reply:
Updated! I'll close this conversation since you have approved. Thank you!